### ISSUE TITLE
Feature wishes

### ISSUE THREAD

**Issue by m7kael** [pixelinfinito/budibase-calendar#9]
Hi! 
I really appreciate the calendar plug in and use it a lot in our BB apps. I have a few wishes for features: 
- Change week start day to monday, currently standard start day is sunday, would it be possible to choose monday?
- Show week number in calendar
- Tooltip when hovering over and event to display another text string
- Change event colour by row value

Once again thanks for creating the plugin and would be super thankful for these features!

**Comment by Sagestum** [comment]
Yes, having an option to set the start day to Monday would be really great. It's very confusing when you're just used to a different calendar view.

**Comment by Sagestum** [comment]
I checked the latest commit and thanks for adding German, unfortunately it does not show up :(
![dropdown](https://github.com/pixelinfinito/budibase-fullcalendar/assets/11883823/92e3d82a-6939-44f3-8b73-c28cee7d1942)
